src/platforms/android/include.gradle

```diff
@@ -1,3 +1,4 @@
 dependencies {
-    compile 'com.tylerjroach:eventsource:1.2.11'   
-}
\ No newline at end of file
+    implementation group: 'com.squareup.okhttp3', name: 'okhttp', version: '3.8.1'
```

Comment:
I would like to see dependencies pulled at build time instead of committed into the repo. If you've got it working with committed libs, it should just be a question of running the right gradle tasks first. Ideally they are already pulled as part of the existing build.